src/dashboard/explorer/explorer_find_capacity.md

```diff
@@ -1,32 +1,97 @@
-# Find Capacity on the Grid
+<h1> ThreeFold Nodes Explorer </h1>
 
-![ ](../img/explorer_basics_.png)
-![ ](../img/explorer_basics_2.png)
+<h2>Table of Contents</h2>
 
-Before you can deploy on the TFGrid you need to find a node which fits your requirements, in terms of location, available resources, ... .
+- [Introduction](#introduction)
+- [Nodes Explorer Links](#nodes-explorer-links)
+- [Nodes Explorer Parameters](#nodes-explorer-parameters)
+- [Nodes Explorer Filters](#nodes-explorer-filters)
+  - [Note on Public IP](#note-on-public-ip)
+- [Finding a Node](#finding-a-node)
+- [Questions and Feedback](#questions-and-feedback)
 
-## Use the explorer you need for your chosen tfgrid net
+***
 
-!!!include:explorer_list
+## Introduction
 
-## Use filter to select Country & Requirement for Public IP's
+In this section, we cover how to use the ThreeFold Nodes Explorer to find capacity on the TFGrid. 
 
-![ ](../img/explorer_find_country_pubip.png)
+Using the ThreeFold Nodes Explorer is a quick way to make sure that you are deploying on a node that is optimal for your specific workload. For example, you can use the Explorer to verify that there is enough capacity on a given node for your workload requirements.
 
-In this case 17 nodes are found with that requirement.
+## Nodes Explorer Links
 
-Do note you have to put
+Each network has a specific Dashboard. Make sure to use the correct URL that corresponds to the network you are deploying on.
 
-- ```>=1``` in the publicip's part, means at least one public ip address.
+- [Devnet Dashboard](https://dashboard.dev.grid.tf)
+  - [Nodes Explorer](https://dashboard.dev.grid.tf/explorer/nodes)
+- [Qanet Dashboard](https://dashboard.qa.grid.tf)
+  - [Nodes Explorer](https://dashboard.qa.grid.tf/explorer/nodes)
+- [Testnet Dashboard](https://dashboard.test.grid.tf)
+  - [Nodes Explorer](https://dashboard.test.grid.tf/explorer/nodes)
+- [Mainnet Dashboard](https://dashboard.grid.tf) 
+  - [Nodes Explorer](https://dashboard.grid.tf/explorer/nodes)
 
-Remember the ID, this is the number you will need to specify your node.
+## Nodes Explorer Parameters
 
-## Important
+There are many ways to use the Nodes Explorer. 
 
-- make sure there is enough capacity on the node for your workload requirements
-- make sure you use the right explorer, each net has a different explorer.
+To search only nodes with GPU, click on the toggle button **GPU Node (Only)**.
 
-## Node details
+![explorer_gpu](../img/explorer_toggle_gpu.png)
 
-![ ](../img/node_detail_.png)
-![ ](../img/node_detail_1.png)
+To search only gateway nodes, click on the toggle button **Gateways (Only)**.
+
+![explorer_gateway](../img/explorer_toggle_gateways.png)
+
+On the right of the window, there is a drop-down menu to filter nodes based on their status: **Up**, **Down** and **Standby**. By default, this is set to **Up**.
```

Comment:
No, the default in all of the filters is now none, so here the default will be to list all the nodes up, down, and standby (so none).